src/app/component/eco-news/components/news-list/news-list.component.ts

```diff
@@ -8,6 +9,7 @@ import * as fromApp from '@store/app.reducers';
 import * as fromEcoNews from '@eco-news-store/eco-news.actions';
 import { EcoNewsSelectors } from '@eco-news-store/eco-news.selectors';
 import { EcoNewsDto } from '@eco-news-models/eco-news-dto';
+import { MatSnackBarComponent } from './../../../errors/mat-snack-bar/mat-snack-bar.component';
```

Comment:
Please, sort imports in such order: angular files, core files/libraries (RxJS etc.), local files. Speaking about this row, please, replace long dotted path with alias

Reply:
fixed